heron/tools/ui/resources/static/js/physical-plan.js

```diff
@@ -86,6 +93,18 @@
       return d.children.length;
     });
 
+    function formatByteSize(byteSize) {
+      if (byteSize > 1024 * 1024 * 1024 / 2) {
+        return (byteSize / (1024 * 1024 * 1024)).toFixed(2) + 'GB';
+      } else if (byteSize > 1024 * 1024 / 2) {
+        return (byteSize / (1024 * 1024)).toFixed(2) + 'MB';
+      } else if (byteSize > 1024 / 2) {
+        return (byteSize / 1024).toFixed(2) + 'KB';
+      } else {
+        return byteSize + 'B';
+      }
+    }
```

Comment:
There is a `pretty-bytes` npm package that does this.

Reply:
Thanks.

Reply:
hmm. maybe future TODO since I don't want to add new dependency now.